source/runner/_tests/fixtures/results/__DangerfileBadSyntax.js.json

```diff
@@ -8,6 +8,6 @@
   "warnings": [],
   "messages": [],
   "markdowns": [
-    "## Error ReferenceError\n```\nhello is not defined\nReferenceError: hello is not defined\n    at Object.<anonymous> (/Users/orta/dev/projects/danger/danger-js/source/runner/_tests/fixtures/__DangerfileBadSyntax.js:4:1)\n    at Module._compile (module.js:624:30)\n    at requireFromString (/Users/orta/dev/projects/danger/danger-js/node_modules/require-from-string/index.js:28:4)\n    at Object.<anonymous> (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:95:21)\n    at step (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:32:23)\n    at Object.next (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:13:53)\n    at /Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:7:71\n    at new Promise (<anonymous>)\n    at __awaiter (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:3:12)\n    at Object.runDangerfileEnvironment (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:66:12)\n```\n### Dangerfile\n```\n1| /*eslint-disable */\n2| \n3| hello\n--^\n4| \n```\n  "
+    "## Error ReferenceError\n```\nhello is not defined\nReferenceError: hello is not defined\n    at Object.<anonymous> (/Users/orta/dev/projects/danger/danger-js/source/runner/_tests/fixtures/__DangerfileBadSyntax.js:4:1)\n    at Module._compile (module.js:624:30)\n    at requireFromString (/Users/orta/dev/projects/danger/danger-js/node_modules/require-from-string/index.js:28:4)\n    at Object.<anonymous> (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:95:21)\n    at step (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:32:23)\n    at Object.next (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:13:53)\n    at /Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:7:71\n    at new Promise (<anonymous>)\n    at __awaiter (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:3:12)\n    at Object.runDangerfileEnvironment (/Users/orta/dev/projects/danger/danger-js/distribution/runner/runners/inline.js:66:12)\n```\n### Dangerfile\n```\n1| hello\n--^\n2| \n```\n  "
```

Comment:
I am not really sure about this one. Tests are passing with and without these changes. As I've removed the `eslint-ignore` from the source file I think the error message should change as well.

Reply:
Ah the tests are still in write mode, I should write a danger rule for that